src/app/services/auth/auth.service.ts

```diff
@@ -1,41 +1,77 @@
 import { Injectable } from '@angular/core';
 import { Account } from '../../models/Account';
+import { User } from '../../models/User';
 import { Router } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { environment } from '../../../environments/environment';
+import { Image } from 'src/app/models/Image';
 
 
 @Injectable({
   providedIn: 'root',
 })
 export class AuthService {
-  
-private loggedIn: boolean = false;
-
-constructor(private router: Router) { }
 
+  //RIGHT NOW WE HAVE A USER AND A ACCOUNT OBJECT
+  //FRONT END IS MAKING TWO CALLS WHEN LOGGING IN
+  //THIS NEEDS TO BE CHANGED
```

Comment:
I'm not actually concerned about this.  If we're worried about performance we can just implement caching.  Also, don't use all caps for comments